### ISSUE TITLE
New items for classes

### ISSUE THREAD

**Issue by ObaniGemini** [Ombridride/minetest-minetestforfun-server#105]
For the moment, classes only have one special item. I think it's not enough to chose a class that to only have a litlle change in stats (armor, health, stamina, mana,...) so I think we should talk about adding some new special items.

For the moment, I just though about battle axes for warriors, throwable hatchets for hunters, and magical powers for wizards.

What would you like to add ?

**Comment by Ombridride** [comment]
Good idea, we wil add this kind of things when classes will be here and __stable__, for the moment it's clearly __not the priority__
